src/process/tags.md

```diff
@@ -0,0 +1,56 @@
+# Process tags
+
+Description of tags, used for projects.
+
+## Status
+- refine `#c377e0` - task already have resolving code, but code needs some fixes.
```

Comment:
@artemiy312 

ooh, so much comments. Let's diccuss it :)

> refine, take it, discuss are not a state

every task has this lifecycle:
idea (needs info) -> requirements (needs info) -> ready to work (take it) -> in progress (no tag) -> review (refine or or discuss nothing)

That's why i call this tags as states. In most cases they are mutually exclusive.
But very clear name for this group is not very important, i think. I just tried to divide tags to groups to avoid big mixed tags list.


**refine** - refine tag is good to mark tasks that is ready for 90%. For example business ask me to estimate milestone. I open it, see tasks list and immediately know, that tasks with refine tag are ready for 90% and throw them out of estimation.
Another often case - someone forgot about his opened PR and his refine task becomes more and more stale. So, when i check everyone's tasks list (i do it every week), i can see faster what refine tasks becomed stale and can mention about it.
I manage project tasks with variations refine tag for three years already. And it's always helps me.

**take it** - y, i thought like you many times.
But in practice if ussue has no tags (blocking too), it was just created by someone and thrown. And it's possible in theory too. Because every teammate with every role can create task. Not every teammate knows well how to work with tags.
In fact this task always is like this: be discussion or good idea for the later time, or bad described bug and so on.
Two-three times a week i receive message from one of our devs to give him next task. And i open 80% of tracker without (or wrong) tags and try to find what can i give. And i spend lots of time to sort tracker when i should not do this.

**needs clarification** - `discuss` have different value. Discuss is smth, we are doing now.
Someone waits answer from opposite side. In fact discuss is blocked task.
And in practice it's convenient to see what task are currently blocked by discussion.

`needs info` marks tasks, that was badly described and leaved in tracker. So, manager can do several things with it:
- mention tasks creator to describe it more comprehensive
- initiate research of this task if it's creator is not available already
- just close it, if task is not so important for additional mention-clarification working

`needs info` is also convenient tag for manager. It shows what tasks needs some additional working before turn to `take it` state. And about naming: phrase `needs clarification` describes good tags meaning too, but `needs info` is also good but have much less symbols.